search/search.tf

```diff
@@ -22,6 +22,16 @@ resource "aws_security_group_rule" "search_ingress_bastion" {
   description       = "ES and Kibana ingress via bastion"
 }
 
+resource "aws_security_group_rule" "search_ingress_cloudplatform" {
+  security_group_id = "${aws_security_group.search_sg.id}"
+  type              = "ingress"
+  protocol          = "tcp"
+  from_port         = 443
+  to_port           = 443
+  cidr_blocks       = ["172.20.0.0/16"]
```

Comment:
Could this CIDR be declared as a var or local somewhere and use that?
I assume it's the cloud platform CIDR?
so ${local.cloud_platform_vpc_cidr} or similar

Reply:
I've updated the code to use a var passed in from env-configs common